Dfe.Academies.Academisation.Data.UnitTest/ProjectAggregate/IncompleteProjectsGetDataQueryTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoFixture;
+using Dfe.Academies.Academisation.Data.ProjectAggregate;
+using Dfe.Academies.Academisation.Data.UnitTest.Contexts;
+using Dfe.Academies.Academisation.Domain.Core.ProjectAggregate;
+using Dfe.Academies.Academisation.Domain.ProjectAggregate;
+using Xunit;
+
+namespace Dfe.Academies.Academisation.Data.UnitTest.ProjectAggregate
+{
+	public class IncompleteProjectsGetDataQueryTests
+	{
+		private readonly Fixture _fixture = new();
+
+		private readonly IncompleteProjectsGetDataQuery _subject;
+		private readonly AcademisationContext _context;
+
+		public IncompleteProjectsGetDataQueryTests()
+		{
+			_context = new TestProjectContext().CreateContext();
+			_subject = new IncompleteProjectsGetDataQuery(_context);
+		}
+
+		[Fact]
+		public async Task ProjectExists___GetProject()
+		{
+			// arrange
+			(ProjectDetails projectDetails1, ProjectState projectState1) = CreateTestProject(region: "region1");
+			(ProjectDetails projectDetails2, ProjectState projectState2) = CreateTestProject("localAuth2");
+			(_, ProjectState projectState3) = CreateTestProject("localAuth3", "region3");
```

Comment:
More for me to understand, why is the discard used in this test? I assume to denote that the project does not exist and should not be returned? Might be worth having a comment or something for the slower people like me :D

Reply:
Project 3 has a region and local authority which means it complete and not required to be changed.
Perhaps I could rename it to complete project? Very open to suggestions on how to make this clearer.

Reply:
Have updated the variable names to make this clearer.